### ISSUE TITLE
Needs review and Azure Pipelines hookup

### ISSUE THREAD

**Issue by CZEMacLeod** [VsixCommunity/Community.VisualStudio.VSCT#1]
@madskristensen I've pushed this as a first draft of the package based on the previous work and discussion at [madskristensen/Community.VisualStudio.Toolkit/#28](https://github.com/madskristensen/Community.VisualStudio.Toolkit/issues/28).
It meets all the criteria as far as I can tell and includes as much documentation as I can for now.

Unfortunately, now that it is part of the VsixCommunity organisation, I cannot update the repo to make it public, or add the webhooks for azure pipelines to do the compile and publish to [nuget.org](nuget.org).

Can you either give me access to the settings on this repo, or take over and do the required appveyor stuff to it?

**Comment by madskristensen** [comment]
Absolutely. You have permissions now. I made the repo public

**Comment by madskristensen** [comment]
I also added the NuGet API key to the org's secrets

**Comment by CZEMacLeod** [comment]
@madskristensen  Not sure I can access the github secrets, but maybe if we switch to GitHub Actions in the future that will be useful.
I am still getting an error with linking azure devops - Unable to configure a service on the selected GitHub repository. This is likely caused by not having the necessary permission to manage hooks for the selected repository.

**Comment by madskristensen** [comment]
I think we should standardize around GitHub Actions or AppVeyor which the toolkit already uses. I can set up automatic AppVeyor CI build and NuGet publishing, but I have zero experience with GH Actions. I don't think we should add Azure Pipelines to the mix, unless there's a specific reason?

**Comment by CZEMacLeod** [comment]
The only reason would be that I use Azure Pipelines for my other projects and thus already know how. 
Ironically, I watched [The .NET Docs Show - The Fastest NuGet Package Ever Published (Probably)](https://www.youtube.com/watch?v=A93Fn_qMLX4) today which shows a template including GitHub Actions so maybe that is the way to go.
I'll have a go at adding that stuff and we can see what happens. It can be a learning experience!

**Comment by madskristensen** [comment]
Yeah, I definitely think GitHub Actions is the way forward. The API key is stored in the secret **NUGET_API_KEY**

**Comment by madskristensen** [comment]
I created a new issue for hooking up AppVeyor so we can get the first package published. Then we are unblocked to integrate it into the project templates.

https://github.com/VsixCommunity/Community.VisualStudio.VSCT/issues/3
